person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name
+  attr_reader :age, :parent_permission
```

Comment:
- [ ] Since both _name_ and _age_ values will change during implementation, these should be marked as attributes accessors.
- [ ] Each person object needs to be identified with a unique id, autogenerated. 
Kindly ensure to improve upon the above, similar to:

```ruby
...
  attr_accessor :name, :age 

  attr_reader :id 
 
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end

...
```